### DISCUSSION TITLE
Apply configuration to group by name

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#31567] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

v38.93.2

### Please tell us more about your question or problem

To draw extra attention to certain licensing issues, we have several presets on a corporate level. They are `packageRules` that force or restrict versions for specific packages. I put the same `groupName` on all of them.

Now, I don't want to force how the individual teams deal with them, but for my own team I'd like to disable the `dependencyDashboardApproval`. Is there a way to set configuration for a group? If not, how else would I put extra configuration on the corporate level presets, without needing to update my team's when the preset change or get extended on a corporate level?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>
/

**Comment by rarkins** [comment]
If you construct your config as nested presets then you might be able to use `ignorePresets`.
